cmds/copy.c

```diff
@@ -56,18 +57,18 @@ static int cmd_cpphfs2phfs(handler_t srcHandler, addr_t srcAddr, size_t srcSz, h
 		srcAddr += res;
 		size -= res;
 
-		if ((res = phfs_write(dstHandler, dstAddr, buff, res)) < 0) {
-			log_error("\nCan't write data to address: 0x%x\n", dstAddr);
+		if ((res = phfs_write(dstHandler, addr, buff, res)) < 0) {
```

Comment:
You may `return res` instead of `return -EIO` below, I think `phfs_write()` should tell the real cause of failure. Especially in the context that `cmd_parse(..)` will return `int` error cause in a few days.

Reply:
Done